sklearn/metrics/scorer.py

```diff
@@ -198,12 +250,18 @@ def __call__(self, clf, X, y, sample_weight=None):
                 elif isinstance(y_pred, list):
                     y_pred = np.vstack([p[:, -1] for p in y_pred]).T
 
+        if self._pass_classes is None:
+            kwargs = self._kwargs
```

Comment:
not covered!

Reply:
maybe i should take this if inside the function. otherwise i'll have to write more tests for all 3 kind of scorers, which seem redundant.